src/state/widgets/selectors.js

```diff
@@ -24,3 +25,27 @@ export const getTypologyWidgetList = createSelector(getListWidget, widgetList =>
     }
     return acc;
   }, {}));
+export const getWidgetInfo = createSelector([getWidgets], (widget) => {
+  const { info } = widget;
```

Comment:
Why not declaring the `getWidgetsInfo` selector and use that, to be coeherent with all the rest?